setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
Actually it is easy to use shebang on Windows.
http://whitescreen.nicolaas.net/programming/windows-shebangs
It's like a polyfill.
I rather develop in mcedit on a server.